### ISSUE TITLE
Consider renaming this repo to reflect its true scope

### ISSUE THREAD

**Issue by TThurston** [EnvironmentAgency/dst-guides#30]
This repo was originally created by one of the pioneering teams in the Environment Agency, and its current name reflects this.  However its scope, and the people using / contributing to it, has since increased.  An updated name may be appropriate.

We should decide its new scope, in particular whether it encompasses all Defra or remains at the Environment Agency level, and propose new names as appropriate.

**Comment by teddmason** [comment]
I'd originally thought that, until I saw that @Cruikshanks had craftily adjusted the documentation to change dst to mean digital service team, rather than south. If that was the cause for concern?

**Comment by Cruikshanks** [comment]
Yep, just following up on @teddmason comment I added commit 4076568 to change the scope from the old Digital Service Team (south) to all Digital Service Teams.

However on whether this should be going across all EA (so non-digital services), or even all Defra I would be reticent to expand it currently until all parties have had an opportunity to go through what we've got and feed in their opinions. We only mentioned yesterday this existed to some of DDTS developers for example.

Perhaps we should be more explicit about the scope in the root README? 💭
